### DISCUSSION TITLE
Add widget to a page

### DISCUSSION THREAD

**Discussion by invaders-xx** [filamentphp/filament#2125] [Help]
Is there a way to add widgets to a page without using getHeaderWidgets and getFooterWidgets function ?

**Comment by ryangjchandler** [comment]
@invaders-xx Take a look at how it's done in `x-filament::widgets`: [packages/admin/resources/views/components/widgets.blade.php](https://github.com/laravel-filament/filament/blob/2.x/packages/admin/resources/views/components/widgets.blade.php)

**Reply by invaders-xx** [reply]
@ryangjchandler thanks ;-)
